Yank/multistate/analyzers/multistateanalyzer.py

```diff
@@ -8,40 +8,48 @@
 Analyze
 =======
 
-Analysis tools and module for YANK simulations. Provides programmatic and automatic "best practices" integration to
-determine free energy and other observables.
+Analysis tools and module for MultiStateSampler simulations. Provides programmatic and automatic
+"best practices" integration to determine free energy and other observables.
 
 Fully extensible to support new samplers and observables.
 
-"""
 
+"""
 
 # =============================================================================================
-# Analyze datafiles produced by YANK.
+# MODULE IMPORTS
 # =============================================================================================
 
 import os
 import abc
 import copy
-import typing
-from typing import Union, Optional
-
 import yaml
+import mdtraj
+import logging
 import numpy as np
-
+import simtk.unit as units
 import openmmtools as mmtools
-from .repex import Reporter
 
-from pymbar import MBAR  # multi-state Bennett acceptance ratio
-from pymbar import timeseries  # for statistical inefficiency analysis
+from . import analyzerutils as autils
```

Comment:
I'd call these extra utility functions module just `utils` so that people can refer to them as `analyzers.utils` and `multistate.utils` instead of `analyzers.analyzerutils` and `multistate.samplingutils`. I understand this creates files with the same name, but I'm not sure if there are downsides for this.